internal/containers/provider/provider.go

```diff
@@ -12,10 +12,14 @@ import (
 )
 
 type ExecConfig struct {
-	// The command to be executed in the provider
+	// The "entrypoint" for the exec command. This is not the docker run entrypoint as that command cannot be overridden
+	// at exec time, but rather a command to be prepended to the Command field before running the test.
+	Entrypoint []string
```

Comment:
I might be missing this, but can you add a test for this leg?

Reply:
I removed this as it wasn't adding a lot of value.